src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,145 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.CustomException;
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationValidator;
+    private static StorageDao storageDao;
+    private static User userIsValid;
+    private static User invalidUser;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationValidator = new RegistrationServiceImpl();
+        storageDao = new StorageDaoImpl();
+        userIsValid = new User();
+        invalidUser = new User();
+    }
+
+    @BeforeEach
+    void setUp() {
+        userIsValid.setId(77L);
```

Comment:
why do we need to set id here to both users?

id is set automatically after registration